cfpb/consumerfinance.gov
# Form explainers: replace image positioning and sizing JS with CSS

Updates Wagtail version of form explainers to use CSS instead of JS for image sizing and positioning.



## Additions

- CSS (flex, viewport units, posiition:sticky) to handle sizing and positioning form explainers images 


## Removals

- Form explainer JS that handles image resizing and positioning


## How to test this PR

1. Check out branch
2. Run gulp
3. In Wagtail admin, create a new Form Explainer page and add an Explainer block.
4. Add a tall image, create some notes, and add another page with a wide image and some notes.
5. Resize screen vertically and horizontally and verify that images resize correctly.
6. Verify that when notes exceed the height of the image, it sticks to the top of the screen correctly when scrolling until notes scroll offscreen.


## Screenshots





## Checklist

  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".




#### Browser testing

Visually tested in the following supported browsers:
